MAVProxy/mavproxy.py

```diff
@@ -1012,11 +1025,10 @@ def open_telemetry_logs(logpath_telem, logpath_telem_raw):
                 mpstate.status.exit = True
                 return
 
-        # use a separate thread for writing to the logfile to prevent
+        # Use a separate thread for writing to the logfile to prevent
```

Comment:
Why would you do this?

Every time I come back in to review this PR I have to look through these all the time.  The patch adds no value to the stated goal of the PR and wastes my time.